src/item/StringToItemParser.php

```diff
@@ -1124,6 +1124,7 @@ private static function make() : self{
 		$result->register("iron_shovel", fn() => VanillaItems::IRON_SHOVEL());
 		$result->register("iron_sword", fn() => VanillaItems::IRON_SWORD());
 		$result->register("jungle_boat", fn() => VanillaItems::JUNGLE_BOAT());
+		$result->register("kelp", fn() => VanillaBlocks::KELP()->asItem());
```

Comment:
```suggestion
		$result->registerBlock("kelp", fn() => VanillaBlocks::KELP());
```
And also it should be below this:
```php
$result->registerBlock("jungle_wooden_stairs", fn() => VanillaBlocks::JUNGLE_STAIRS());
```